pkg/psmdb/container.go

```diff
@@ -59,6 +62,10 @@ func container(cr *api.PerconaServerMongoDB, replset *api.ReplsetSpec, name stri
 			MountPath: "/etc/users-secret",
 		})
 	}
+	hostPort := int32(0)
```

Comment:
what happens for version 1.12? Is it OK to set host port to 0?

Reply:
I think it's ok, because the default value for hostPort before 1.12 is also 0